web-extensions/chrome/engine/datamodel.js

```diff
@@ -246,6 +246,21 @@ BmcComic.prototype.serialize = function() {
     };
 }
 
+/**
+ * This method loops over sources.
+ *
+ * @param {object} callback - The callback to be called on each source.
+ *
+ */
+BmcComic.prototype.iterSources = function(callback) {
```

Comment:
Thought I had added it Oo ?! Looks like I somehow forgot to add it despite that... my bad !